src/classes/CRLP_Operation_TEST.cls

```diff
@@ -55,4 +55,112 @@ private class CRLP_Operation_TEST {
         System.assertEquals(CRLP_Operation.RollupType.SUM, CRLP_Operation.getRollupTypeFromString('Sum'));
     }
 
+    /**
+     * @description CRLP_Operation.RollupType should be in one-to-one correspondce with Rollup__mdt.Operation__c Picklist Values.   Each RollupType values' name should be a Rollup__mdt.Operation__c Picklist Value, and each Rollup__mdt.Operation__c Picklist Value should have a corresponding CRLP_Operation.RollupType whose name equals the Picklist Value
+     */
+    @IsTest
+    private static void assertRollupTypeExistsForEachRollupOperationPicklistValue() {
+        Set<String> enumNames = new Set<String>();
+        for (CRLP_Operation.RollupType rollupType : CRLP_Operation.RollupType.values()) {
+            enumNames.add(rollupType.name());
+        }
+
+        Test.startTest();
+
+        // CRLP_Operation
+        {
```

Comment:
Personal Preference.  I use empty curly brackets to provide scope.  I've gotten into the habit when testing to start an opening curly bracket to denote the test class I'm testing in case a Test Class tests more than one class.  

Similarly, I include another empty bracket for (1) scope and (2) to denote a method or class I'm testing.

The scope reason is so I never reuse vars when testing, so I never forget to "reset" a var before a test.